src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/OpprettBehandling/OpprettBehandling.tsx

```diff
@@ -61,27 +59,23 @@ const OpprettBehandling: React.FC<IProps> = ({ behandling, fagsak, onListElement
                     }}
                 >
                     <Modal.Body>
-                        <FamilieSelect
-                            erLesevisning={true}
+                        <Select
+                            readOnly={true}
                             name={'Behandling'}
                             label={'Type behandling'}
-                            value={skjema.felter.behandlingstype.verdi}
-                            lesevisningVerdi={
-                                behandlingstyper[Behandlingstype.REVURDERING_TILBAKEKREVING]
-                            }
+                            value={Behandlingstype.REVURDERING_TILBAKEKREVING}
                         >
-                            {behandlingsTyper.map(opt => (
+                            {Object.values(Behandlingstype).map(opt => (
```

Comment:
`behandlingsTyper` var en liste med de to verdiene som finnes for `Behandlingstype` så bruker heller enumen direkte